app/models/concerns/spree/subscription_plan/role_subscriber.rb

```diff
@@ -1,11 +1,11 @@
 module Spree
-  class Subscription < Spree::Base
+  class SubscriptionPlan < Spree::Base
     module RoleSubscriber
       extend ActiveSupport::Concern
 
       included do
-        before_create :add_role_subscriber, :unless => :unsubscribed_at?
-        before_update :remove_role_subscriber, :if => :unsubscribed_at?
+        # before_create :add_role_subscriber, :unless => :unsubscribed_at?
```

Comment:
and even if we have commented this, why the methods(add_role_subscriber and remove_role_subscriber) are still present?